extension/src/experiments/index.ts

```diff
@@ -600,6 +600,21 @@ export class Experiments extends BaseRepository<TableData> {
     return experiment?.id
   }
 
+  private getOldestCommitHashFromData(data: ExperimentsOutput) {
```

Comment:
Is there a better way to find the oldest commit hash?

Reply:
I think it would be safe to assume that the last one is the oldest. From the docs: 
```
Experiments in the table are first grouped (by parent commit). 
They are then sorted inside each group, chronologically by default. 
The --sort-by and --sort-order options can change this ordering, based on any single, visible metric or param.
```

```
--sort-order {asc,desc} - sort order to use with --sort-by. Defaults to ascending (asc).
```

```
-n <num>, --num <num> - show experiments from the last num commits (first parents) starting from the --rev baseline. 
Give a negative value to include all first-parent commits (similar to git log -n).
```

Reply:
> I think it would be safe to assume that the last one is the oldest. From the docs:

The experiment data is an object though and I thought the order isn't guaranteed in JS objects 🤔 I'll go ahead and update the code for now though!